composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
i'm afraid this change will be a deal breaker.

<img width="406" alt="screen shot 2016-06-30 at 7 30 13 pm" src="https://cloud.githubusercontent.com/assets/53660/16510231/216e7fc6-3ef9-11e6-8911-f1407074416f.png">